qiskit/opflow/primitive_ops/tapered_pauli_sum_op.py

```diff
@@ -122,6 +125,17 @@ def __init__(
         self._sq_paulis = sq_paulis
         self._sq_list = sq_list
         self._tapering_values = tapering_values
+        self._tol = tol
+
+    @property
+    def tol(self):
+        """Tolerance threshold for ignoring real and complex parts of a coefficient."""
+        return self._tol
+
+    @tol.setter
+    def tol(self, value):
+        """Set the tolerance threshold for ignoring real and complex parts of a coefficient."""
+        self._tol = value
```

Comment:
What's the reason for adding a property to wrap the normal behaviour?  It doesn't seem to be doing anything except adding overhead.

Reply:
I added it for consistency with `tapering_values` which is the only other attribute that is settable. Personally I'd also prefer to just have it as public attribute (just `self.tol` in the init) 🙂

Reply:
From the user's perspective there wouldn't be any inconsistency unless they're in the habit of requiring `del x.tol` to raise an error or other weird stuff.  But also, I don't really care haha.